docs/running.md

```diff
@@ -9,14 +9,224 @@ a file `$XDG_STATE_HOME/warnet/warnet.log`, otherwise it will use `$HOME/.warnet
+
+`just p` - forwards port from local into the cluster
+
+<details>
+    <summary>port forward details</summary>
+
+    ```
+    kubectl port-forward svc/rpc 9276:9276
+    ```
+
+</details>
+
+### Running the commands directly
+
+You can setup the cluster following the steps below, if you do not wish to use the `justfile` option above.
+
+```bash
+# Creates a Kubernetes namespace for the RPC
+kubectl apply -f src/templates/rpc/namespace.yaml
+
+# Applies Role-Based Access Control (RBAC) configuration for the RPC
+kubectl apply -f src/templates/rpc/rbac-config.yaml
+
+# Deploys a service to expose the RPC
+kubectl apply -f src/templates/rpc/warnet-rpc-service.yaml
+
+# Deploys a statefulset for the RPC to manage stateful applications
+kubectl apply -f src/templates/rpc/warnet-rpc-statefulset.yaml
+
+```
+
+### Setup and Start RPC in Development Mode with Minikube
+
+The commands below sets up and starts the RPC service in development mode using Minikube. The steps are as follows:
+
+```bash
+# Mounts the local source directory to the Minikube virtual machine, allowing the RPC service to access code and files from the local development environment.
+minikube mount $PWD:/mnt/src > /tmp/minikube_mount.log 2>&1 &
+
+# Capture the PID of the minikube mount command
+MINIKUBE_MOUNT_PID=$!
+
+# Save the PID to a file for later use
+echo $MINIKUBE_MOUNT_PID > /tmp/minikube_mount.pid
+
+# Setup k8s
+kubectl apply -f src/templates/rpc/namespace.yaml
+kubectl apply -f src/templates/rpc/rbac-config.yaml
+kubectl apply -f src/templates/rpc/warnet-rpc-service.yaml
+kubectl apply -f src/templates/rpc/warnet-rpc-statefulset-dev.yaml
+kubectl config set-context --current --namespace=warnet
+
+# Wait for the RPC pod to become ready, ensuring that the service is fully operational before proceeding.
+sleep 2
+kubectl wait --for=condition=Ready --timeout=2m pod rpc-0
+
+```
+
+### Stop RPC in Development Mode with Minikube
+
+Follow the steps below to stop the RPC service running in development mode using Minikube:
 
 ```bash
-kubectl apply -f '*.yaml'
+# Deletes the Kubernetes namespaces 'warnet' and 'warnet-logging', which contain the resources associated with the RPC service and its logging, respectively.
+kubectl delete namespace warnet
+kubectl delete namespace warnet-logging
+
+# Sets the current Kubernetes context to the default namespace, ensuring that subsequent Kubernetes commands operate within this namespace.
+kubectl config set-context --current --namespace=default
+
+# Fetch job ID of `minikube mount $PWD:/mnt/src` from saved PID file
+if [ -f /tmp/minikube_mount.pid ]; then
+    MINIKUBE_MOUNT_PID=$(cat /tmp/minikube_mount.pid)
+    # Stop the background job using its PID
+    kill -SIGINT $MINIKUBE_MOUNT_PID
+    # Optionally, removes the PID file
+    rm /tmp/minikube_mount.pid
+else
+    echo "PID file not found. Minikube mount process may not have been started."
+fi
+
 ```
 
+### Setup and Start RPC in Development Mode with Docker Desktop
```

Comment:
I'm not sure this section and the next deal with Docker Desktop. That said, I have not used Docker Desktop yet. So, maybe I'm wrong about this??